Makefile

```diff
@@ -9,17 +8,20 @@ test_files =$(wildcard tests/**/*.ts)
 test_out_files = $(test_files:tests/%.ts=tests/%.lua)
 test_out_init = $(test_init:tests/%.ts=tests/%.lua)
 
-compile:
+install:
+	npm ci
```

Comment:
**issue**: Short names are harder to understand for those who are not used to these commands

**suggestion**: change to `npm clean-install`